go-controller/pkg/node/healthcheck.go

```diff
@@ -25,6 +25,7 @@ type loadBalancerHealthChecker struct {
 	server    healthcheck.Server
 	services  map[ktypes.NamespacedName]uint16
 	endpoints map[ktypes.NamespacedName]int
+	sync.Mutex
```

Comment:
@trozet can you put this at the top of the struct? style nit.

Reply:
ok